website/src/data/users.js

```diff
@@ -136,6 +136,15 @@ const users = [
     fbOpenSource: false,
     pinned: false,
   },
+  {
+    title: 'Console Table',
+    description: 'Printing Pretty Tables on your console.',
+    preview: require('./showcase/console-table.png'),
+    website: 'https://console-table.netlify.app/',
```

Comment:
FYI @lex111 I think it's fine to accept Netlify domains like that, as long as it's a "production/stable" domain, without a random deployment number. It's a free hosting solution and not worst than accepting GH pages domains